application/forms/Command/Instance/ToggleInstanceFeaturesForm.php

```diff
@@ -37,6 +45,61 @@ public function __construct($featureStatus)
         ];
 
         $this->getAttributes()->add('class', 'instance-features');
+
+        $this->on(self::ON_SUCCESS, function () {
+            /** @var ToggleInstanceFeatureCommand $feature */
+            foreach ($this->submittedFeatures as $feature) {
+                switch ($feature->getFeature()) {
+                    case ToggleInstanceFeatureCommand::FEATURE_ACTIVE_HOST_CHECKS:
+                        $message = tp(
+                            'Enabled active host checks successfully',
+                            'Disabled active host checks successfully',
+                            (int) $feature->getEnabled()
+                        );
```

Comment:
Mhm, what? Doesn't even work I suppose, does it? Don't use `tp` as control structure :wink:

Reply:
Sure, it does work. Don't know what the problem is, I am passing the states as int, this isn't a control structure.

Reply:
It is. `tp` means `translatePlural` and is used to designate different message versions. One for singular and one for plural. You use it to choose two entirely different messages. That's a mis-use of `tp`.